compiler/ballerina-parser/src/test/resources/misc/resiliency/resiliency_source_05.bal

```diff
@@ -0,0 +1 @@
+var x = from var a in m = {
```

Comment:
Do we run this test? Seems the test  is not added to the `ResiliencyTest` class

Reply:
Thanx